python/lsst/dax/imgserv/config/imgserv_api_v1.json

```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#", 
+  "id": "https://lsst.org/schemas/imgserv_api_v1.json#",
+  "type": "record",
+  "properties": {
+    "namespace": "lsst.dax.imgserv",
+    "name": "imageserv_API",
+    "description": "ImageServ API v1 Schema",
+    "version": "1"
+  },
+  "required": [ "name", "namespace", "version" ],
+  "definitions": 
+  {
+    "rectangle": {
+      "type": "object",
+      "properties": {
+        "width": { "type": "double" },
+        "height": { "type": "double" },
+        "unit": { "enum": [ "pixel", "arcsec" ] },
+        "ucd": "pos.outline;obs" 
+      },
+      "required": [ "width", "height", "unit", "ucd" ] 
+    },
+    "circle": {
+      "type": "object",
+      "properties": {
+        "unit": { "type": { "enum": [ "deg" ] } },
+        "ucd": "pos.outline;obs",
+        "radius": { "type": "double" }
+     },
+      "required": [ "unit", "ucd", "radius" ]
+    },
+    "range": {
+      "type": "object",
+      "properties": {
+        "long1": { "type": "double" },
+        "lat1": { "type": "double" },
+        "long2": { "type": "double" },
+        "lat2": { "type": "double" }
+      },
+     "required": [ "long1", "lat1", "long2", "lat2" ]
+    },
+    "polygon": {
+      "type": "object",
+      "properties": {
+        "unit": { "type": { "enum": [ "deg" ] } },
+        "ucd": "pos.outline;obs",
+        "sides": {
+          "type": "array",
+          "minimum": 3,
+          "multipleOf": { "type": "double" }
+        },
+        "required": [ "unit", "ucd", "sides" ]
+      }
+   },
+   "filter": { 
```

Comment:
I only commented on indentation when it didn't match the surrounding indentation. "polygon" and "range" above is indented 4 spaces, "filter" is indented 3 spaces. (counting from the "+")